dhcpv4/bsdp/boot_image.go

```diff
@@ -99,3 +110,37 @@ func (b *BootImage) Unmarshal(buf *uio.Lexer) error {
 	b.Name = string(buf.Consume(int(nameLength)))
 	return buf.Error()
 }
+
+func getBootImageID(code dhcpv4.OptionCode, o dhcpv4.Options) *BootImageID {
+	v := o.Get(code)
+	if v == nil {
+		return nil
+	}
+	var b BootImageID
+	if err := uio.FromBigEndian(&b, v); err != nil {
```

Comment:
Should this be `b.FromBytes(v)` instead of `uio.FromBigEndian(&b, v)`?

Reply:
Could be, but b.FromBytes(v) is just uio.FromBigEdnain(&b, v) - no difference really